### PR TITLE
Add s3 config versioning

### PR DESCRIPTION
**Pull request by dominickendrick**
## What does this change?

This PR add support for versioning of our config file via the new updates in `configuration-magic`.
- Pin our config to different versions via the `s3ConfigVersion` variable
- Add a read me entry on the manual process for updating config
- There is a new updated config file at `eu-west-1-frontend.v1.conf` that adds Targetting tool config options:
  - `aws.composer.account.role`
  - `target.campaigns.bucket`
## What is the value of this and can you measure success?

We can now version our config! This allows us to:
- test our changes locally
- track and version changes in config, whilst keeping them private.
## Does this affect other platforms - Amp, Apps, etc?

No
## Request for comment

@TBonnin @currysoup @guardian/dotcom-platform

### CODE REVIEW

File: docs/03-dev-howtos/13-Update-configuration-in-s3.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+This contains all the configuration for each application and for each stage.
+
+To add or update a configuration item you need to:
+- Download the configuration file
+- This is located at
+```
+https://s3-eu-west-1.amazonaws.com/aws-frontend-store/config/eu-west-1-frontend.conf
+```
+
+- This can be done by hand via the console using the [aws cli tools](http://docs.aws.amazon.com/cli/latest/userguide/installing.html)
```
Comment (gtrufitt) [inline docs/03-dev-howtos/13-Update-configuration-in-s3.md]: Should this be *or* using the aws cli...

**Comment by gtrufitt** [comment]
Make sense to me 👍

**Comment by TBonnin** [comment]
👍

**Comment by johnduffell** [comment]
thanks dom! 👍 🚀

**Pull request merged**